ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -488,11 +497,15 @@ class ServicesHomeScreen extends React.Component<Props, State> {
           title: I18n.t("services.title"),
           body: () => <Markdown>{I18n.t("services.servicesHelp")}</Markdown>
         }}
+        isSearchAvailable={this.props.userMetadata !== undefined}
```

Comment:
Could you leave a comment explaining the meaning of that condition?

Reply:
added to the main component description within https://github.com/teamdigitale/io-app/pull/1338/commits/4f6eb900a7db5adb0a7a43ea2af861eec0bb6c9b